### PR TITLE
Fix additional reviewers to min and latest dependency checker

### PR DESCRIPTION
**Pull request by gsheni**
- Both workflows are failing on main due to a syntax error:
  - https://github.com/alteryx/featuretools/actions/runs/1068139279
  - https://github.com/alteryx/featuretools/actions/runs/1068139296
- This MR fixes it by referencing the correct team.

### CODE REVIEW

File: .github/workflows/latest_dependency_checker.yml [hunk 1]
```diff
@@ -31,4 +31,4 @@ jobs:
         branch: latest-dep-update
         branch-suffix: short-commit-hash
         base: main
-        team-reviewers: @alteryx/ml-tools
+        team-reviewers: alteryx/ml-tools
```
Comment (rwedge) [inline .github/workflows/latest_dependency_checker.yml:34]: based on how team-reviewers is used in [this example](https://github.com/peter-evans/create-pull-request#reference-example), do you think just `ml-tools` would work?

Comment (gsheni) [reply .github/workflows/latest_dependency_checker.yml]: Fixed. I also lowered the pandas version so we can test this.

**Comment by rwedge** [comment]
to test, could we change the github action to happen on push to that branch?

**Comment by gsheni** [comment]
No, we can't. The PR won't be generated on push because there won't be a diff with `main`

**Comment by gsheni** [comment]
@rwedge No, we can't. The PR won't be generated on push because there won't be a diff with `main`

**Review by rwedge — APPROVED** [review]
LGTM

**Pull request merged**